Makefile

```diff
@@ -8,7 +8,7 @@ ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
 VERSION:=$(shell awk -F\' '/const VERSION/ {print $$2}' < src/DDTrace/Tracer.php)
 
-INI_FILE := /usr/local/etc/php/conf.d/ddtrace.ini
+INI_FILE := $(shell php -i | grep 'Scan' | sed -e 's/Scan this dir for additional .ini files =>//g' | head -n 1 | awk '{print $$1}')/ddtrace.ini
```

Comment:
That is cleaner - thanks! And good point about `(none)/ddtrace.ini`. Since the Makefile is used for our containers only, do you think we need to come up with a solution to the `(none)` issue? Or do you think it's OK to leave as is?

Reply:
I imagine users that build from source may use the `Makefile` too. So maybe we need to address it?

Reply:
Building from source with PECL or manually will replace the Makefile when `phpize` is run so they won't encounter this issue.